ui-v2/tests/acceptance/dc/services/show-routing.feature

```diff
@@ -0,0 +1,37 @@
+@setupApplicationTest
+Feature: dc / services / Show Routing for Serivce
+Scenario: Given a service, the Routing tab should display
+    Given 1 datacenter model with the value "dc1"
+    And 1 node models
+    And 1 service model from yaml
+    ---
+    - Service:
+        Kind: connect

```

Comment:
I think this is a typo here. It doesn't matter too much as it's not a 'Kind' that hides a tab. But I think it should be `Kind: consul`